parmed/openmm/parameters.py

```diff
@@ -151,6 +151,12 @@ def write(self, dest, provenance=None, write_unused=True):
             templates remaining and parameters including those atom types will
             not be written. A ParameterWarning is issued if any such residues are
             found in a).
+        ljforce : bool
+            If True will turn off L-J and use the LennardJonesForce to use
+            CostumNonbondedForce to compute L-J interactions. Default is False. It
+            should be set to True when converting a charmm force field file that doesn't
+            have NBFIX in the file so that the ffxml conversion is compatible with the
+            main charmm36.xml file
```

Comment:
Add a note that when pair-specific L-J modifications are present (e.g., NBFIX in CHARMM), this behavior is always done and this flag is ignored.